unity-renderer/Assets/DCLPlugins/ECS7/ComponentCrdtWriteSystem/ComponentCrdtWriteSystem.cs

```diff
@@ -127,7 +127,7 @@ private bool DispatchSceneMessages(string sceneId, Queue<CRDTMessage> messages)
 
     private void DispatchBinaryMessage(string sceneId, MemoryStream stream)
     {
-        WebInterface.SendBinaryMessage(sceneId, stream.ToArray());
+        RPCGlobalContext.context.crdtContext.notifications.Enqueue((sceneId, stream.ToArray()));
```

Comment:
Not sure if ServiceLocator is the proper place, since it's just data.
Maybe DataStore is a more suitable place, but I don't feel very comfortable adding there, since at this point we have no idea about how many data types are we going add to that context, so it might become an asmdef nightmare moving it there